packages/web-components/fast-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+  accentFillRestBehavior,
+  heightNumber,	
+  neutralForegroundRestBehavior
+} from "../styles/index";
+
+export const AvatarStyles = css`
+  ${display("flex")} 
+  :host {
+    max-width: calc(${heightNumber} * 2.5px);
+    height: calc(${heightNumber} * 2.5px);
+    position: relative;
+  }
+
+  .link {
+    text-decoration: none;
+    color: var(--neutral-foreground-res);
```

Comment:
We'll want to interpolate the color variables behaviors here and point to them such as [this example](https://github.com/microsoft/fast/blob/1cdf9ef352d6a0f1c385badc8d3255e9c0a78cc6/packages/web-components/fast-components/src/divider/divider.styles.ts#L12).